tools/hacked/splitFasta/splitFasta.py

```diff
@@ -0,0 +1,14 @@
+import sys, os
+if __name__ == "__main__":
+    #assuming perfect input, read every two lines
+    inpath = sys.argv[1]
+    file_contents = open(inpath, 'r').readlines()
+    os.makedirs('splits')
+    inname = os.path.basename(inpath)
+    for i in range(0, len(file_contents), 2):
+        headline = file_contents[i]
+        outname = headline[1:headline.index(' ')]+'.fa'
+        outfile = open(os.path.join('splits',outname), 'w')
+        outfile.write(file_contents[i])
+        outfile.write(file_contents[i+1])
+        outfile.close()
```

Comment:
@erasche what do you think about this code:
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/sequence.py#L424

Without the biopython dependency ...